client/python/gradio_client/utils.py

```diff
@@ -926,7 +926,7 @@ def _json_schema_to_python_type(schema: Any, defs) -> str:
     type_ = get_type(schema)
     if type_ == {}:
         if "json" in schema.get("description", {}):
-            return "Dict[Any, Any]"
+            return "str | list | dict"
```

Comment:
I guess we could collapse both cases into `str | bool | int | float | list | dict` since that covers the spectrum of possible json payloads

Reply:
updated